concepts/05 UI Components/DataGrid/05 Accessibility/10 Accessibility Standards Compliance.md

```diff
@@ -21,14 +29,53 @@ The DataGrid component meets a variety of <a href="https://www.access-board.gov/
         <td>2.1.1 Keyboard (Level A)</td>
         <td><ul>
                 <li>Keyboard navigation is not supported for ColumnChooser items in 'drag' mode.</li>
+                <li>DataGrid is navigated with the Tab key in the data area.</li>
                 <li>The master-detail feature does not support keyboard navigation.</li>
                 <li>Adaptive rows cannot be expanded/collapsed using keyboard navigation.</li>
             </ul>    
         </td>
     </tr>
-        <tr>
+    <tr>
+        <td>-</td>
+        <td>2.4.11 Focus Not Obscured (Minimum) (Level AA 2.2 only)</td>
+        <td>ColumnChooser and HeaderFilter in DataGrid do not meet this criterion.</td>
+    </tr>
+    <tr>
+        <td>-</td>
+        <td>2.5.7 Dragging Movements (Level AA 2.2 only)</td>
+        <td>
+            <ul>
+                <li>ColumnChooser in 'drag' mode.</li>
+                <li>Grouping and local reordering.</li>
```

Comment:
What is local reordering? I suspect it's drag and drop rows. Maybe you can call it that